api/src/main/java/net/luckperms/api/platform/Platform.java

```diff
@@ -90,4 +98,28 @@
             return this.friendlyName;
         }
     }
+
+    /**
+     * Represents the environment LuckPerms is being run on.
+     */
+    enum Environment {
+        CLIENT("Client"),
```

Comment:
When would it ever be necessary to run LuckPerms on a client?

Reply:
I should clarify that this is more likely refers to the type of server (an integrated server) rather than a physical client (since LP is bound to a server or proxy no matter what).